packages/gui/src/components/nfts/NFTPreview.tsx

```diff
@@ -47,8 +293,18 @@ export type NFTPreviewProps = {
   elevate?: boolean;
   background?: any;
   hideStatusBar?: boolean;
+  isPreview?: boolean;
+  metadata?: any;
+  isOffer?: boolean;
```

Comment:
The property name should reflect the behavior that it defines, not the origin of the property. Since the goal is to prevent displaying the thumbnail when viewing an offer, I would suggest a name like `disableThumbnail`

Reply:
Done.